src/sage/combinat/all.py

```diff
@@ -277,7 +277,7 @@
 lazy_import('sage.combinat.binary_recurrence_sequences',
             'BinaryRecurrenceSequence')
 lazy_import('sage.combinat.recognizable_series', 'RecognizableSeriesSpace')
-lazy_import('sage.combinat.k_regular_sequence', 'kRegularSequenceSpace')
+lazy_import('sage.combinat.k_regular_sequence', 'RegularSequenceRing')
```

Comment:
I propose to rename the file `k_regular_sequence.py` to `regular_sequence.py` (after merging #36001) for consistency: now that everything is renamed in such a way to remove the variable `k`, that would seem to be a logical move to me.

Reply:
Agreed.

Reply:
Yes, indeed, I agree that this seems a logical move. I prepared a commit and will push it soon. (BTW it does not seem to be a problem due to #36001 when merging; `git` appearently takes care of this.)